### PR TITLE
update site for fall 2021

### PR DESCRIPTION
**Pull request by n4t3m**
i wonder if someone will read this

### CODE REVIEW

File: layouts/default.vue [hunk 1]
```diff
@@ -76,10 +76,10 @@
           <div class="three wide column">
             <h4 class="ui inverted header"> Join the Community!</h4>
             <div class="ui inverted link list">
+              <a class="item" href="https://discord.com/invite/vESWWrm"><i class="icon discord"></i> Discord</a>
               <a class="item" href="https://ucrcyber.slack.com"><i class="icon slack"></i> Slack</a>
               <a class="item" href="https://linkedin.com/company/cyberucr"><i class="icon linkedin"></i> LinkedIn</a>
               <a class="item" href="https://linkedin.com/company/cyberucr"><i class="icon instagram"></i> Instagram</a>
```
Comment (AnthonyH45) [inline layouts/default.vue]: Please fix the links here, I see two for linked in, re-add the instagram link

File: pages/calendar.vue [hunk 2]
```diff
@@ -4,14 +4,14 @@
     <div class="ui middle aligned stackable grid container">
       <div class="row">
         <div class="eight wide left column">
-          <h3 class="ui header">Meetings: Winter 2020</h3>
+          <h3 class="ui header">Meetings: Fall 2021</h3>
           <!-- <p> General meetings cover announcements, defensive/offensive demos, tech-talks, and guest speakers. We also host weekly practices for Defensive and Offensive competitions! Anyone is free to join. Email <b> cyberucr@gmail.com </b> for more info. </p>  -->
           <p> This Winter we're hosting biweekly intermediate-leveled General Meetings alongside defensive and offensive security workshop meetings. Download our Google calendar to yours! </p>
```
Comment (AnthonyH45) [inline pages/calendar.vue]: You fixed the header to say Fall, but need to fix the description to reflect that

File: layouts/default.vue (hunk 1 above)
Comment (n4t3m) [reply layouts/default.vue]: issue has been fixed

File: pages/calendar.vue (hunk 2 above)
Comment (n4t3m) [reply pages/calendar.vue]: issue has been fixed

File: layouts/default.vue (hunk 1 above)
Comment (AnthonyH45) [reply layouts/default.vue]: Ty

File: pages/calendar.vue (hunk 2 above)
Comment (AnthonyH45) [reply pages/calendar.vue]: Ty

**Pull request merged**